### PR TITLE
Adding StorSimple Hyak specification to generate .NET sdk

### PR DESCRIPTION
**Pull request by avirupch**

### CODE REVIEW

**Comment by ogail** [comment]
add to whitelist

File: src/StorSimple/Microsoft.WindowsAzure.Management.StorSimple.nuspec [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <references>
+      <group targetFramework="portable-net45+sl50+wp80+win">
+        <reference file="Microsoft.WindowsAzure.Management.StorSimple.dll" />
+      </group>
+      <group targetFramework="net40">
+        <reference file="Microsoft.WindowsAzure.Management.StorSimple.dll" />
+      </group>
+    </references>
+    <dependencies>
+      <dependency id="Microsoft.WindowsAzure.Common" version="[1.4.0,2.0)" />
```
Comment (stankovski) [inline src/StorSimple/Microsoft.WindowsAzure.Management.StorSimple.nuspec]: All new packages should take a dependency on Microsoft.Azure.Common `[2.0.1, 3.0)`

File: src/StorSimple/packages.config [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Generator" version="1.0.5478.27459-prerelease" targetFramework="portable-net45+sl50+win+wp80" />
+  <package id="Microsoft.WindowsAzure.Management.StorSimple.Specification" version="1.0.5484.40347-prerelease" targetFramework="portable-net45+sl50+win+wp80" />
+  <package id="Microsoft.Bcl" version="1.1.9" targetFramework="portable-net45+sl50+win+wp80" />
+  <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="portable-net45+sl50+win+wp80" />
+  <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="portable-net45+sl50+win+wp80" />
+  <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.9.10826.1824" targetFramework="portable-net45+sl50+win+wp80" />
+  <package id="Microsoft.Net.Http" version="2.2.22" targetFramework="portable-net45+sl50+win+wp80" />
+  <package id="Microsoft.WindowsAzure.Common" version="1.4.0" targetFramework="portable-net45+sl50+win+wp80" />
```
Comment (stankovski) [inline src/StorSimple/packages.config]: Update this to take dependency on Microsoft.Azure.Common version 2.0.1

File: src/StorSimple/packages.config [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  <package id="Microsoft.WindowsAzure.Common.Dependencies" version="1.1.0" targetFramework="portable-net45+sl50+win+wp80" />
```
Comment (stankovski) [inline src/StorSimple/packages.config]: Update this to Microsoft.Azure.Common.Dependencies version 1.0.0

File: src/StorSimple/packages.config [hunk 4]
```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Generator" version="1.0.5478.27459-prerelease" targetFramework="portable-net45+sl50+win+wp80" />
+  <package id="Microsoft.WindowsAzure.Management.StorSimple.Specification" version="1.0.5484.40347-prerelease" targetFramework="portable-net45+sl50+win+wp80" />
```
Comment (stankovski) [inline src/StorSimple/packages.config]: Add Hyak.Common version 1.0.1

File: src/StorSimple/StorSimple.csproj [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+    <Compile Include="Generated\StorSimpleManagementClient.cs" />
+    <Compile Include="Generated\StorSimpleManagementClientExtensions.cs" />
+    <Compile Include="Generated\VirtualDiskOperations.cs" />
+    <Compile Include="Generated\VirtualDiskOperationsExtensions.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="hyak.xml" />
+  </ItemGroup>
+  <ItemGroup>
```
Comment (stankovski) [inline src/StorSimple/StorSimple.csproj]: Replace this entire ItemGroup with:
```
<ItemGroup>
    <Reference Include="Hyak.Common">
      <HintPath>..\..\packages\Hyak.Common.1.0.1\lib\portable-net403+win+wpa81\Hyak.Common.dll</HintPath>
    </Reference>
    <Reference Include="Microsoft.Azure.Common">
      <HintPath>..\..\packages\Microsoft.Azure.Common.2.0.1\lib\portable-net45+wp8+wpa81+win\Microsoft.Azure.Common.dll</HintPath>
    </Reference>
  </ItemGroup>
```

**Pull request merged**